bin/node/runtime/src/lib.rs

```diff
@@ -1972,7 +1972,21 @@ impl_runtime_apis! {
 
 	impl pallet_nomination_pools_runtime_api::NominationPoolsApi<Block, AccountId, Balance> for Runtime {
 		fn pending_rewards(member_account: AccountId) -> Balance {
-			NominationPools::pending_rewards(member_account).unwrap_or_default()
+			NominationPools::api_pending_rewards(member_account).unwrap_or_default()
+		}
+
+		fn points_to_balance(pool_id: u32) -> Result<Balance, ()> {
```

Comment:
What is the best practice to handle errors in runtime APIs? Should this method instead return `Balance` and handle potential errors with `unwrap_or_default()` as above in the `api_pending_rewards` method? I could not find any documentation on this.